build.md

```diff
@@ -8,7 +8,7 @@ You can build the Spacemesh Full Node and CLI wallet directly from source code,
 
 The Spacemesh App which includes the full p2p node and a wallet is supported on any Linux distro that supports [Electron Apps](https://electronjs.org/docs/tutorial/support).
 
-If you are running a Linux that doesn't support Electron than you can build and run a full p2p node and a CLI wallet.
+If you are running a Linux distribution that doesn't support Electron then you can build and run a full p2p node and a CLI wallet.
```

Comment:
I don't understand: if the Linux distro does not support Electron, is it OK or not?

Reply:
It is not OK for running the Spacemesh App. In this case you need to build go-spacemesh + cli wallet from code and use them together to participate in the testnet. Hopefully, we'll have this config ready for Spacemesh0.1 - see the DoD sheet being worked on...

Reply:
Shall we change "you can build" to "you need to build"?

Reply:
fine with me.